thoth/common/openshift.py

```diff
@@ -1505,6 +1505,7 @@ def schedule_mi_workflow(
         repository: str,
         entities: Optional[str] = None,
         knowledge_path: Optional[str] = None,
+        mi_thoth: bool = False,
```

Comment:
what is this variable used for?  do you mean like `mi_used_for_thoth`?

Reply:
true, commited your suggestion